src/puffin/CMakeLists.txt

```diff
@@ -18,11 +18,10 @@
 set(ICEBERG_PUFFIN_SOURCES demo_puffin.cc)
 set(ICEBERG_PUFFIN_INCLUDES "${ICEBERG_API_DIR}")
 
-ADD_ICEBERG_LIB(
-  iceberg_puffin
-  CMAKE_PACKAGE_NAME
-  puffin
-  SOURCES
-  ${ICEBERG_PUFFIN_SOURCES}
-  PRIVATE_INCLUDES
-  ${ICEBERG_PUFFIN_INCLUDES})
+add_iceberg_lib(iceberg_puffin
```

Comment:
Might be personal preference but I find the original formatting more readable. I don't like this much indentation. Do you think there is a way to configure that?

Reply:
I simply used the `cmake-format` plugin in vscode with default setting which formatted these CMake files.

Reply:
> I simply used the `cmake-format` plugin in vscode with default setting which formatted these CMake files.

Yeah, but we need to check the format when PRs are created, I did not include the CI part because I not sure what is the best way to do it. We might consider that together with clang-format.